pkgs/by-name/li/libvisual/package.nix

```diff
@@ -2,16 +2,21 @@
   lib,
   stdenv,
   fetchFromGitHub,
-  fetchpatch,
   SDL,
   autoreconfHook,
   autoconf-archive,
   glib,
   pkg-config,
+  fullVariant ? false,
+  withLvTool ? fullVariant,
+  withExamples ? fullVariant,
```

Comment:
It would be simpler to just have the 2 flags and turn them both on in the full variant but if the examples are not even installed then it is kinda useless to compile them and if the lv-tools are not functional at all, why do we have an option for them?